tests/controller/util.test.ts

```diff
@@ -0,0 +1,110 @@
+import { LAMPORTS_PER_SOL, SystemProgram, Transaction } from "@solana/web3.js";
```

Comment:
I think the file name should be solanaHelper.test.ts as we are doing unit test on the file